src/quest.coffee

```diff
@@ -226,6 +227,15 @@ class Quest
     else
       console.log "Nothing to output.".gray
 
+  jsonify: (sqlResult, options) ->
+    console.log JSON.stringify row, 4 for row in sqlResult.rows
```

Comment:
Just call `JSON.stringify` on `sqlResult.rows` which will output it as a json list of objects.